gbpservice/nfp/config_orchestrator/agent/topics.py

```diff
@@ -10,9 +10,9 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-FW_NFP_CONFIGAGENT_TOPIC = 'oc-firewall-agent'
-LB_NFP_CONFIGAGENT_TOPIC = 'oc-lbaas_agent'
-VPN_NFP_CONFIGAGENT_TOPIC = 'vpn_agent'
+FW_NFP_CONFIGAGENT_TOPIC = 'nfp-firewall-agent'
```

Comment:
Can we discuss this along with Ahmed? Topic changes needs to be done for other party too. Are they already done?